x-pack/test/functional/apps/snapshot_restore/home_page.ts

```diff
@@ -0,0 +1,31 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import expect from '@kbn/expect';
+import { KibanaFunctionalTestDefaultProviders } from '../../../types/providers';
+
+// eslint-disable-next-line import/no-default-export
+export default ({ getPageObjects, getService }: KibanaFunctionalTestDefaultProviders) => {
+  const pageObjects = getPageObjects(['common', 'snapshotRestore']);
+  const log = getService('log');
+
+  describe('Home page', function() {
+    this.tags('smoke');
+    before(async () => {
+      await pageObjects.common.navigateToApp('snapshotRestore');
+    });
+
+    it('Loads the app', async () => {
+      const appTitle = 'Snapshot Repositories';
+      await log.debug(`Checking for app title to be {appTitle}`);
+      const appTitleText = await pageObjects.snapshotRestore.appTitleText();
+      expect(appTitleText).to.be(appTitle);
+
+      const repositoriesButton = await pageObjects.snapshotRestore.registerRepositoryButton();
```

Comment:
(similar to previous) https://github.com/elastic/kibana/pull/39193 was merged, which changed the default tab to Snapshots instead of Repositories. I think this assertion is still valid, because we show the register repository button on empty snapshot list as well, but might want to double check

Reply:
Fixed